### ISSUE TITLE
Alien-like sound from inferenced audio at loss rate ~ 0.8

### ISSUE THREAD

**Issue by amarbayar** [espnet/espnet#5960]
**Describe your question**

I am training a TTS model using Fastspech2. I started training my model about 6 hours ago (40,000~ steps) and loss rate dropped from `4~` down to `0.8~`. I tried running an inference and got this: https://voca.ro/194j5BQAX8ZP (and bunch more from the validation dataset all sounded like this "alien" 👽

My steps:
- Created my recipe in the `egs2` dir by cloning an existing recipe
- Prepared my dataset (about 6 hours worth of dataset, 6000~ hq audio files and transcriptions) 
- Trained a Tacotron2 model to generate durations and completed that 
- Then triggered my Fastspeech2 model training

At loss rate of about `1.4` I tried inference and the audio files sounded like  "alien" but I thought it was too early to try. But now that it is down to `0.8` I was wondering if I should start seeing improvements but the output seems to be similar. 

One thing I noticed was that when I did the teacher-student forcing, it generated wav files in:
- `exp/tts_train_tacotron2_raw_char/decode_use_teacher_forcingtrue_train.loss.ave/dev/wavs`
- `exp/tts_train_tacotron2_raw_char/decode_use_teacher_forcingtrue_train.loss.ave/test/wavs`
- `exp/tts_train_tacotron2_raw_char/decode_use_teacher_forcingtrue_train.loss.ave/train/wavs`

The inferenced audio files sound almost similar to these audio wavs that got generated way early in the pipeline stage. 

Could this step have messed up my training data so training/inferences are also getting messed up should I wait?

**Comment by sw005320** [comment]
Can you paste the learning curve?
Can you train more?

**Comment by amarbayar** [comment]
@sw005320  --- Here are the graphs. I stopped the training after 17 hours because the inference wasn't producing intelliglbe results at all. And now I trying to see if I can retrain my Tacotron2 (teacher) again to generate durations and then retrain Fastspeech2. The retraining of Tacotron2 has just plateaued around 0.5. 

<img width="668" alt="Screenshot 2024-11-21 at 5 23 24 PM" src="https://github.com/user-attachments/assets/cb4b5845-6ada-4fa2-8c5d-fdf7382ee036">
<img width="673" alt="Screenshot 2024-11-21 at 5 23 06 PM" src="https://github.com/user-attachments/assets/c5e173d2-620a-4038-8cba-042aa8907e10">
<img width="668" alt="Screenshot 2024-11-21 at 5 22 57 PM" src="https://github.com/user-attachments/assets/aad3d8b2-dd70-404a-a9e5-d286f0465a26">
<img width="669" alt="Screenshot 2024-11-21 at 5 22 04 PM" src="https://github.com/user-attachments/assets/ce5376ff-a421-4a85-9c94-8014ad2c1440">
<img width="668" alt="Screenshot 2024-11-21 at 5 21 50 PM" src="https://github.com/user-attachments/assets/fba1338e-b818-4e3e-85cf-28eb7544b756">
<img width="679" alt="Screenshot 2024-11-21 at 5 21 30 PM" src="https://github.com/user-attachments/assets/1e155cac-245a-4785-bffb-d784a31aa95f">

<img width="673" alt="Screenshot 2024-11-21 at 5 23 33 PM" src="https://github.com/user-attachments/assets/9529e034-0e0e-4bc1-9039-24a506405c25">
<img width="678" alt="Screenshot 2024-11-21 at 5 23 49 PM" src="https://github.com/user-attachments/assets/74f1c26c-b86e-4a06-ba2a-8d2293545d60">

**Comment by ftshijt** [comment]
There are two key issues that might be the cause:
- the training time is still not enough. Unfortunately, the loss in TTS is in general not a good indicator and it would usually be beneficial to increase the training time even after the loss is stable (good to get to around 100k at least)
- The output wave is likely to be from the vocoder. Can you please explain more about which vocoder you are using?
